### DISCUSSION TITLE
How to make cursor bigger?

### DISCUSSION THREAD

**Discussion by HubKing** [ihhub/fheroes2#6530] [Q&A] [answered]
Wayland + Gnome. I had been using fractional scaling but it caused too much problems, so I switched to 100% scaling + font scaling. It seems that fheroes does not respect font scaling, and the cursor is very small.

Is there any way to set scaling for the cursor?

**Comment by oleg-derevenetz** [comment, marked as answer]
> It seems that fheroes does not respect font scaling

And it is extremely unlikely that it will do so in the foreseeable future. GUI elements of the original game (taken from the original AGG files) are designed for specific font sizes, not with scalable fonts in mind.

> Is there any way to set scaling for the cursor?

Try to enable the software emulated cursor via `fheroes2.cfg`. Software emulated cursor should adapt its size to the in-game resolution.

**Reply by HubKing** [reply]
I did not expect in-game font to be scaled by that setting, but the cursor size. Anyway, where is that file and how should I enable software cursor? I installed fheroes2 using AUR Flatpak. I found the location of it under `/var/lib/flatpak`, but I could not see the cfg file.

**Reply by oleg-derevenetz** [reply]
I don't use Flatpak myself, but the config file should be somewhere in your home directory, because it's, naturally, per-user. In Linux, `fheroes2.cfg` is usually located in the `$XDG_CONFIG_HOME/fheroes2` (usually, `~/.config/fheroes2`), but for Flatpak this path may be different. Try to look somewhere in the `~/.var/app/io.github.ihhub.Fheroes2` directory.

**Reply by HubKing** [reply]
Yeah, I had looked into the `~/.config` directory before writing the reply above, but I could not find one, so I thought maybe the file is inside of the Flatpak. Following your answer, I found the file at `~/.var/app/io.github.ihhub.Fheroes2/config/fheroes2/fheroes2.cfg`. I changed `cursor soft rendering` to `on`, and now the cursor size is correct. In my opinion, this should have been the default option, because the game itself does not scale according to the native resolution, and probably users are using a lower resolution (in my case,  864x486).
